python/ray/tests/test_channel.py

```diff
@@ -1033,12 +1025,6 @@ def write(self, value):
         # are the same actor. Note that reading the channel multiple
         # times is supported via channel cache mechanism.
         ray.get(actor1.read.remote())
-    """
```

Comment:
Add tests for these two cases

Reply:
So I did add these two test cases in test_accelerated_dag.
test_driver_and_actor_as_readers does 1, and test_multiple_reads_from_same_actor does 2.

But I can add them also as specific channel tests if needed.

Reply:
Tests in `test_accelerated_dag.py` are end-to-end tests for RayCG. This test focuses on channels. Adding tests here would be helpful, as it simplifies debugging compared to debugging an end-to-end test.

Reply:
added test for 2, how would I test composite channels for 1, it's reads through cached channel right if there's two tasks from the same actor and test_cached_channel_multi_readers does that